slf4j-api/src/main/java/org/slf4j/helpers/BasicMDCAdapter.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2004-2011 QOS.ch
+ * Copyright (c) 2004-2011 QOS.ch, Copyright (C) 2015 Google Inc.
```

Comment:
This the first time I encounter this issue. I don't fully understand the implications, just enough to be uncomfortable.